pep-0001.txt

```diff
@@ -278,24 +278,28 @@ for the discussion, so long as the the following criteria are met:
 * A direct link to the current discussion thread is provided in the PEP
   under the ``Discussions-To`` header.
 
-Typically, the `Python-Dev`_ mailing list and the
-`PEPs category`_ of the `Python Discourse`_ are good choices for most PEPs,
-while some specialized topics have specific venues, such as
+Typically, the `PEPs category`_ of the `Python Discourse`_
+is a good choice for most new PEPs,
+whereas historically the `Python-Dev`_ mailing list was commonly used.
+Some specialized topics have specific venues, such as
 `Typing-SIG`_ for typing PEPs or the `Packaging category`_ on the Python
 Discourse for packaging PEPs. If the PEP authors are unsure of the best venue,
 the PEP Sponsor and PEP editors can advise them accordingly.
 
-If the chosen venue is not the `Python-Dev`_ mailing list,
-a brief announcement should be posted there when the draft PEP is
-committed to the repository, which should include a link to the rendered PEP
+If the chosen venue is not a Discourse category,
```

Comment:
This is actually a key point that I meant to bring up, thanks. @encukou , any guidance on this?

Historically, in the special case of PEPs that had, per SC permission, their canonical discussion location in another Discourse category (i.e. Packaging), at least recently they've generally not been also posted to Python-Dev (and never the PEPs category), and to require this seemed somewhat noisy, redundant and carries the risk of splitting the discussion between different threads on the very same platform (without Discourse admin intervention to immediately lock the thread each time). Additionally, those just looking for notifications of new PEPs being posted can subscribe to the PEP RSS feed, etc.

However, if @encukou and the SC would like this worded differently, or there's otherwise PEP editor consensus, then I'm of course happy to oblige.

Reply:
Thanks; I ended up rewriting this to consolidate these two mentions of posting on the PEPs category if the discussion thread is not there into one, which made it many lines shorter, as well as being specific about it being the PEPs category as requested.